src/main/resources/schemas/default.xml

```diff
@@ -161,9 +161,21 @@
     <tag name="EPILOGUE" depreciated="Use BACKMATTER with DIV instead of EPILOGUE" where="primary">
       <desc><![CDATA[]]></desc>
     </tag>
-    <tag name="FONTGROUP" where="oldspelling">
+    <tag name="FONT" where="oldspelling">
       <desc><![CDATA[This tag allows the editor to fine-tune font size where necessary, as, for example, on a title page.]]></desc>
-      <attribute name="n" type="select" default="3">
```

Comment:
I've left `FONTGROUP` with `@n` just below, deprecated.  What's your feeling on deprecating vs dropping? It seems there are a few other cases where we really could just drop things completely (eg. `BLOCKQUOTE`, `DIV/@link`) but you've only deprecated them in the past.